docs/channel-line-flex.md

```diff
@@ -130,7 +130,6 @@ async function App(context) {
 > **Note: When it comes to complicated Flex Messages, it is recommended to check your code by the Flex Message Simulator.**
 >
 > - [Flex Message Simulator](https://developers.line.biz/console/fx/)
```

Comment:
When I opened this URL in the browser, I immediately got redirected to `https://developers.line.biz/flex-simulator/`. Should we use `https://developers.line.biz/flex-simulator/` instead the original link?

Reply:
yes